src/main/kotlin/com/exactpro/th2/common/schema/message/impl/rabbitmq/custom/MessageConverter.kt

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2020-2021 Exactpro (Exactpro Systems Limited)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.exactpro.th2.common.schema.message.impl.rabbitmq.custom
+
+/**
+ * Converter for delivery to [T] object and from [T] to delivery.
+ * The delivery is received/sent from/to message queue.
+ */
+interface MessageConverter<T : Any> {
```

Comment:
Please add limitations for class T.

Reply:
What is the bound should be? I don't think that we should limit T in any other way. Essentially, we should not limit it with some interface from Protobuf library